templates/platforms/android-studio/app/build.gradle.kts.hbs

```diff
@@ -61,7 +61,7 @@ dependencies {
     {{~#each android-app-dependencies}}
     implementation("{{this}}"){{/each}}
     implementation("androidx.webkit:webkit:1.4.0")
-    implementation("androidx.appcompat:appcompat:1.5.1")
+    implementation("androidx.appcompat:appcompat:1.4.1")
```

Comment:
The latest stable version is 1.5.0, which is why you were having trouble pulling 1.5.1

Reply:
yeah that's my bad, I was copying the new versions from a project that has been updated to latest versions and I must've forgotten to also update the patch version